website/content/community/maintainers.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Maintainers
+weight: 1
+---
+
+MetalLB was created by Dave Anderson.  We are thankful for his efforts to
+create MetalLB as a valuable contribution to the Kubernetes ecosystem.  Ongoing
+MetalLB development and maintenance is run by a team of current maintainers.
+
+MetalLB maintainers are participants in the project with the ability to approve
+changes to the MetalLB code base.  The current list of maintainers can be found
+in the [MetalLB OWNERS
+file](https://github.com/metallb/metallb/blob/main/OWNERS).
+
+## Maintainer Expectations
+
+An active maintainer should:
+
+* Actively participate in reviewing pull requests and incoming issues.  Note
+  that there are no hard rules on what is "active enough" and this is left up
+  to the judgement of the current group of maintainers.
+
+* Actively participate in discussions about design and the future of the
+  project.
+
+* Take responsibility for backports to appropriate branches for PRs they approve
+  and merge.
+
+* Do their best to follow all code, testing, and design conventions as
+  determined by consensus among active maintainers.
+
+* Gracefully step down from their maintainership role when they are no longer
+  planning to actively participate in the project.
+
+## Adding New Maintainers
+
+New maintainers are added by consensus among the current group of maintainers.
+This can be done via a private discussion via Slack or email.  A majority of
+maintainers should support the addition of the new person, and no single
+maintainer should object to adding the new maintainer.
+
+## Removing Maintainers
+
+It is normal for maintainers to come and go based on their other
+responsibilities.  Inactive maintainers may be removed if there is no
+expectation of ongoing participation.  It should be made very easy to re-add a
```

Comment:
>It should be made very easy to re-add a previous maintainer if they are able to actively participate again.

I agree, though I'm not sure what the purpose of this sentence is in the context of this document.

Reply:
I was trying to make it clear that removing someone for this reason really should not seem like a big deal.  It's based on some of my past experience where people are very hesitant to let go of a maintainership position, though it shouldn't be that way.  Instead of this sentence, an alternative could be something like:

"If a previously inactive maintainer resumes participation, they should be given quick consideration for re-adding to the team."

or I can just drop it if you still don't like calling this out.

Reply:
>If a previously inactive maintainer resumes participation, they should be given quick consideration for re-adding to the team.

I think this ^ conveys the message better. I'd maybe change "previously inactive" to "former" though.

Reply:
done